Modules/Axis/Sources/Axis/String/Collection.swift

```diff
@@ -0,0 +1,16 @@
+extension Collection where Iterator.Element: Equatable, Iterator.Element == SubSequence.Iterator.Element {
+    func index(of part: Self) -> Index? {
```

Comment:
Is this implementation taken from the standard library? Can we get a couple unit tests for this method?